modules/trustxBidAdapter.js

```diff
@@ -7,6 +7,7 @@ import { config } from '../src/config.js';
 const BIDDER_CODE = 'trustx';
 const ENDPOINT_URL = 'https://sofia.trustx.org/hb';
 const NEW_ENDPOINT_URL = 'https://grid.bidswitch.net/hbjson?sp=trustx';
```

Comment:
should this be ssp=?

Reply:
all good with sp=trustx, this is part of the bidding endpoint that is already up & running for a could of pub integrations, so we're all set

thanks for the review!